dbms/src/Flash/Pipeline/Schedule/Tasks/Impls/EventTask.cpp

```diff
@@ -37,7 +37,14 @@ EventTask::EventTask(
 
 void EventTask::finalizeImpl()
 {
-    doFinalizeImpl();
+    try
+    {
+        doFinalizeImpl();
```

Comment:
https://github.com/pingcap/tiflash/blob/5b5355cdb97da1471692208a3f12d990fc98e3c8/dbms/src/Flash/Pipeline/Schedule/Tasks/Task.cpp#L169-L176
<img width="1020" height="534" alt="企业微信截图_85e6196e-3b03-499f-be0b-f29626bfb0c5" src="https://github.com/user-attachments/assets/599f0bde-9e4c-4b46-8deb-df0acf71b5a0" />
But finalizeImpl has been try catch.

Reply:
And if the suffix fails, the Task should also be marked as failed, not ignored.

Reply:
1. Even `finalizeImpl()` has been try catch, but I think should also catch inside `EventTask::finalizeImpl(), to make sure `onTaskFinish()` is always called.
2. Some Operators do prepare work in `Operator::operatorPrefixImpl()`, like `ExchangeSenderSinkOp`, it will call `DAGResponseWriter::prepare()`. I think we should also consider catch exception for `PipelineExec::executePrefix()`. Otherwise we may still got hang issue